dex/src/main/scala/com/wavesplatform/dex/queue/LocalMatcherQueue.scala

```diff
@@ -20,7 +20,7 @@ class LocalMatcherQueue(settings: Settings, store: LocalQueueStore, time: Time)
 
   private val executor = Executors.newSingleThreadExecutor {
     new ThreadFactoryBuilder()
-      .setDaemon(true)
+      .setDaemon(false)
```

Comment:
Is this the reason of not working ctrl+c?

Reply:
Nope, actually, I didn't tried reproduce this problem before moving to coordinated shutdown